src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobBuilder.cs

```diff
@@ -652,24 +651,24 @@ public unsafe void WriteBytes(byte* buffer, int byteCount)
                 Throw.InvalidOperationBuilderAlreadyLinked();
             }
 
-            WriteBytesUnchecked(buffer, byteCount);
+            WriteBytesUnchecked(new ReadOnlySpan<byte>(buffer, byteCount));
         }
 
-        private unsafe void WriteBytesUnchecked(byte* buffer, int byteCount)
+        private void WriteBytesUnchecked(ReadOnlySpan<byte> buffer)
```

Comment:
What does the "Unchecked" in the name refer to now?

Reply:
It does not do `if (!IsHead) { Throw.InvalidOperationBuilderAlreadyLinked(); }`.